pkg/store/sandbox/sandbox.go

```diff
@@ -88,5 +102,7 @@ func (s *Store) List() []Sandbox {
 func (s *Store) Delete(id string) {
 	s.lock.Lock()
 	defer s.lock.Unlock()
+	id, _ = s.idIndex.Get(id) // nolint: errcheck
```

Comment:
One last comment. It's my fault.
I didn't know that `idIndex.Delete` doesn't handle truncated index.
If so, I'm fine with your original code, the new one seems fragile. Sorry for the trouble... :(

Reply:
Both make sence. When i write it at first I just do it like this. We do best effort to clear ID  even through there are some abnormal case in idindex. Because it is regular ID not truncated index in most case. We only use truncindex with crictl.Do you think about it?

Reply:
@yanxuean If there is an error, the returned id will be empty string, I feel like we should just return as what you did before. Please add comment to explain why we return there.

As for the `Delete`, I think it's fine to just call `s.idIndex.Delete(id), we don't need `_ =` in front.

Reply:
done